src/sentry/static/sentry/app/views/settings/incidentRules/triggers/form.tsx

```diff
@@ -34,97 +32,130 @@ type Props = {
    */
   error?: {[fieldName: string]: string};
   projects: Project[];
+  resolveThreshold: UnsavedIncidentRule['resolveThreshold'];
+  thresholdType: UnsavedIncidentRule['thresholdType'];
   trigger: Trigger;
   triggerIndex: number;
   isCritical: boolean;
 
   onChange: (trigger: Trigger, changeObj: Partial<Trigger>) => void;
+  onThresholdTypeChange: (thresholdType: AlertRuleThresholdType) => void;
 };
 
+const CriticalIndicator = styled(CircleIndicator)`
+  background: ${p => p.theme.red400};
+  margin-right: ${space(1)};
+`;
+
+const WarningIndicator = styled(CircleIndicator)`
+  background: ${p => p.theme.yellow400};
+  margin-right: ${space(1)};
+`;
+
+const ResolvedIndicator = styled(CircleIndicator)`
+  background: ${p => p.theme.green400};
+  margin-right: ${space(1)};
+`;
+
 class TriggerForm extends React.PureComponent<Props> {
-  getThresholdKey = (
-    type: AlertRuleThreshold
-  ): AlertRuleThresholdKey[AlertRuleThreshold] =>
-    type === AlertRuleThreshold.RESOLUTION ? 'resolveThreshold' : 'alertThreshold';
+  // Overwritten in ResolvedTriggerForm
+  getThresholdType = () => {
+    return this.props.thresholdType;
+  };
+
+  getTriggerLabel = (): React.ReactNode => {
+    const {isCritical} = this.props;
+    const triggerLabel = isCritical ? t('Critical Status') : t('Warning Status');
+    const TriggerIndicator = isCritical ? CriticalIndicator : WarningIndicator;
+    return (
+      <React.Fragment>
+        <TriggerIndicator size={12} />
+        {triggerLabel}
+      </React.Fragment>
+    );
+  };
+
+  getFieldHelp = (): React.ReactNode => {
+    const {isCritical} = this.props;
+    return tct('The threshold that will activate the [severity] status.', {
+      severity: isCritical ? t('critical') : t('warning'),
+    });
+  };
 
   /**
    * Handler for threshold changes coming from slider or chart.
    * Needs to sync state with the form.
    */
-  handleChangeThreshold = (type: AlertRuleThreshold, value: ThresholdControlValue) => {
+  handleChangeThreshold = (value: ThresholdControlValue) => {
     const {onChange, trigger} = this.props;
 
-    const thresholdKey = this.getThresholdKey(type);
-
     onChange(
       {
         ...trigger,
-        [thresholdKey]: value.threshold,
-        thresholdType: value.thresholdType,
+        alertThreshold: value.threshold,
       },
-      {[thresholdKey]: value.threshold}
+      {alertThreshold: value.threshold}
     );
   };
 
   render() {
-    const {disabled, error, trigger, isCritical} = this.props;
-    const triggerLabel = isCritical
-      ? t('Critical Trigger Threshold')
-      : t('Warning Trigger Threshold');
-    const resolutionLabel = isCritical
-      ? t('Critical Resolution Threshold')
-      : t('Warning Resolution Threshold');
+    const {disabled, error, trigger, isCritical, onThresholdTypeChange} = this.props;
+
+    return (
+      <Field
+        label={this.getTriggerLabel()}
```

Comment:
e.g. here you would have something like `label={this.props.triggerLabel}` and move `getTriggerLabel` up a component.